containers/ecr-viewer/src/app/components/Filters.tsx

```diff
@@ -0,0 +1,266 @@
+"use client";
+
+import React, { useCallback, useEffect, useState } from "react";
+import { Button } from "@trussworks/react-uswds";
+import { useRouter, usePathname, useSearchParams } from "next/navigation";
+
+const basePath =
+  process.env.NODE_ENV === "production" ? process.env.NEXT_PUBLIC_BASEPATH : "";
```

Comment:
`NEXT_PUBLIC_BASEPATH` was replaced with `BASE_PATH` in #2996 

When you merge in main you'll need to change this:
```suggestion
  process.env.NODE_ENV === "production" ? process.env.BASEPATH : "";
```